arch/PKGBUILD

```diff
@@ -0,0 +1,59 @@
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+
+pkgname=tarantool-gis-git
+_pkgname=gis
+pkgver=0.1.r11
+pkgrel=1
+pkgdesc='A full-featured geospatial extension for Tarantool'
+arch=(i686 x86_64)
+url='http://github.com/tarantool/gis'
+license=(BSD)
+#depends=(readline ncurses zlib libyaml)
+makedepends=(cmake tarantool geos proj python-sphinx)
+conflicts=(tarantool-gis)
+provides=(tarantool-gis)
+replaces=(tarantool-gis)
+#source=(https://github.com/tarantool/$_pkgname/archive/$pkgver.tar.gz)
+source=(git+https://github.com/tarantool/$_pkgname)
+
+pkgver() {
+  cd gis
+  git describe | sed 's/^v//; s/-/.r/; s/-.*//'
+}
+
+build() {
+  #cd $_pkgname-$pkgver
+  cd $_pkgname
+
+  cmake \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DCMAKE_INSTALL_BINDIR=/usr/bin \
+    -DCMAKE_INSTALL_SYSCONFDIR=/etc \
+    -DCMAKE_INSTALL_LIBDIR=/usr/lib \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_SKIP_RPATH=ON \
+    -DENABLE_BUNDLED_LIBYAML:BOOL=OFF \
+    -DENABLE_BACKTRACE:BOOL=ON \
```

Comment:
There are no -DENABLE_BACKTRACE and -DENABLE_BUNDLED_LIBYAML in this sub-projects. Is it copy-pasted from [tarantool/tarantool](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=tarantool-git)?